### DISCUSSION TITLE
coroot node agent integration with docker swarm

### DISCUSSION THREAD

**Discussion by Mehdi-BR** [coroot/coroot#247] [Q&A] [answered]
Hello,

we have a docker swarm cluster running on multiple nodes, and we have a monitoring virtual machine, in the monitoring vm we have coroot install as docker compose, and we have coroot node agent installed in all docker swarm nodes as container and pointing to the coroot instance, the agents are recognized by coroot, however not all the docker containers are showing, upon accessing the logs of one of the agents I found these errors:
![image](https://github.com/coroot/coroot/assets/56741513/e8f6cb79-d205-494a-932f-bea1e0c1a388)
So I thought I needed to install cilium in docker swarm, however I found that docker swarm unfortunately is not supported in cilium.
https://github.com/cilium/cilium/issues/4159

**Comment by def** [comment, marked as answer]
It's not an issue, these messages are just for debug purposes.

**Reply by Mehdi-BR** [reply]
So it's supposed to work without cilium, if I understand correctly. Plus, should I add the node agent in the Prometheus configuration in order for it to get the pods?
